packages/design-system/src/locale/lang/en.ts

```diff
@@ -29,7 +29,7 @@ export default {
 	'codeDiff.replacing': 'Replacing...',
 	'codeDiff.undo': 'Undo',
 	'betaTag.beta': 'beta',
-	'askAssistantButton.askAssistant': 'Ask Assistant',
```

Comment:
Did you check this with Nik? [His latest feedback](https://www.notion.so/n8n/e60bfc535a8a49eca03ae512e6b9588b?v=c45eedfcf6474b3798d41ad882aa032f&p=3db9b5fdc7d7444ebd3507f5753878ce&pm=s) was that the buttons should say `Ask Assistant`